Directory.Build.props

```diff
@@ -4,6 +4,7 @@
     <MicrosoftVisualStudioComponentModelHostPackageVersion>17.0.77-g1cdf89c995</MicrosoftVisualStudioComponentModelHostPackageVersion>
     <MicrosoftVisualStudioExtensibilityHostingPackageVersion>17.0.32-g9d6b2c6f26</MicrosoftVisualStudioExtensibilityHostingPackageVersion>
     <MicrosoftVisualStudioPackageCurrent>17.0.0-previews-2-31405-002</MicrosoftVisualStudioPackageCurrent>
+    <MicrosoftVisualStudioCompositionConfigurationVersion>17.0.12-alpha</MicrosoftVisualStudioCompositionConfigurationVersion>
```

Comment:
Had to pin the version to mitigate build errors: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=8050458&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=146f9b6c-b8c6-5b2f-c16b-fe2886abe6ff